aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -9,7 +9,7 @@ uid: tutorials/web-api-help-pages-using-swagger
 ---
 # ASP.NET Core web API help pages with Swagger / OpenAPI
 
-By [Christoph Nienaber](https://twitter.com/zuckerthoben) and [Rico Suter](http://rsuter.com)
+By [Christoph Nienaber](https://twitter.com/zuckerthoben) and [Rico Suter](https://rsuter.com)
```

Comment:
This one redirects ...

```suggestion
By [Christoph Nienaber](https://twitter.com/zuckerthoben) and [Rico Suter](https://blog.rsuter.com/)
```

Reply:
@guardrex - I can change this one. The scope of this User Story (linked in opening comment) is to clear CATS issues by fixing broken links, metadata, and http to https, so quite a few of them may redirect. See the other PRs in the User Story.